api/egl.xml

```diff
@@ -2924,6 +2926,12 @@
                 <enum name="EGL_CUDA_DEVICE_NV"/>
             </require>
         </extension>
+        <extension name="EGL_NV_n_buffer" supported="egl">
```

Comment:
According to the spec, there are two different extensions in the single document, each supporting a single enum. If that's how the driver exposes support for this functionality, then there should be two \<extension> blocks named accordingly - AFAICT there's no actual extension named "EGL_NV_n_buffer". It's fine if the extension .txt document is named that but not the header interfaces.

Reply:
The new parts look fine, but I believe this \<extension> block should be removed.

Reply:
Missed out during merge step, now fixed with latest patches.